File: Easy/FizzBuzz.cpp
vector<string> solve(int n)
{
    vector<string> ans;
    string x = "Fizz", y = "Buzz", z = "FizzBuzz";
    for (int i = 1; i <= n; i++)
    {
        if (i % 3 == 0 && i % 5 == 0)
            ans.push_back(z);
        else if (i % 3 == 0)
            ans.push_back(x);
        else if (i % 5 == 0)
            ans.push_back(y);
        else
            ans.push_back(to_string(i));
    }

    return ans;
}

File: Easy/FizzBuzz/FizzBuzz.js
class Solution {
    solve(n) {
        var fizzbuzz = [];
        var temp = "";
        for (var i = 1; i <= n; i++) {
            // check if the number is divisible by 3 and 5
            if (i % 3 === 0 && i % 5 === 0) {
            // add the word fizzbuzz to the array
            fizzbuzz.push('FizzBuzz');
            } else if (i % 3 === 0) {
            // add the word fizz to the array
            fizzbuzz.push('Fizz');
            } else if (i % 5 === 0) {
            // add the word buzz to the array
            fizzbuzz.push('Buzz');
            } else {
            // add the number to the array
            temp = i.toString();
            fizzbuzz.push(temp);
            }
        }
        // return the array
        return fizzbuzz;
    }
}